docs/api.md

```diff
@@ -2912,6 +2913,9 @@ puppeteer.launch().then(async browser => {
 });
 ```
 
+#### frame.asElement()
```

Comment:
JSHandle.asElement is a conditional cast. Could be replaced by isElement : boolean and nothing except some ergonomics would change.

Your method is very different. It doesn't cast a frame to a different type, it returns an entirely new handle that happens to be associated with the frame.

We should think a lot about what to call this method. `parentElement`, `frameElement` or `elementHandle` come to mind. But I don't really like any of them. The API should be clear about whether is the `<iframe>` element in the parent document, or the `document.documentElement` of the inner contents of the frame.

Reply:
How about `getFrameOwner`? 
We would honor the CDP method. But I don't know if it's a friendly name for the user.

Reply:
getFrameOwner makes me think its going to return the parent frame.

Reply:
hm, I like the `asElement` here - I don't think users of JSHandle.asElement() care much what happens under the hood (and waether we actually want them to care about it).

Reply:
Another argument: `JSHandle.asElement` return synchronously, where as this returns a promise.

Wdyt about `frame.element()`?

Reply:
I like `frame.element`, so we are consistent with `element.contentFrame`. It's not `element.asContentFrame` there.

What do you think @aslushnikov?

Reply:
Hmm seems good to me

Reply:
@pavelfeldman suggests `frame.ownerElement()`. Let's roll!

Reply:
@aslushnikov done!